etna/feedback/forms.py

```diff
@@ -84,12 +84,34 @@ def clean_url(self) -> Union[str, None]:
         }
 
         # Add derived values to 'cleaned_data'
+        path = normalize_path(parse_result.path)
+        path_list = path[1:].split("/")
+        page_title = "None"
+        page_type = "None"
+
+        if path_list[0] == "search":
+            page_type = "Search landing page"
+            if path_list[1]:
+                page_type = f"{path_list[1].title()} search"
+        elif path_list[0] == "catalogue":
+            if iaid := path_list[2]:
+                if iaid[0] == "a" and "-" not in iaid:
+                    page_type = "Archive details page"
+                elif iaid[0] == "f" and "-" not in iaid:
+                    page_type = "Record creator page"
+                else:
+                    page_type = "Record details page"
+                page_title = f"Catalogue IAID: {iaid.upper()}"
```

Comment:
Hi @jamesbiggs, I think the derivations for page_type, page_title for search and various record details could be difficult to maintain. We should try if possible to come up with another way to limit or completely remove hardcoding for search/details pages. Maybe something like the wagtail page attribute? Lets give it some thought/discuss.